routes/activities.js

```diff
@@ -15,6 +15,11 @@ router.get('/', (req, res) => {
 
 router.get('/:id', (req, res) => {
   const { id } = req.params
+  if (isNaN(id)) {
```

Comment:
NaN is a special number value: the string 'foo' is not a number, but it isn't a NaN either.

you may follow this pattern instead: https://github.com/rocambille/back-to-mvc/blob/91b743e6d6e999aba7ee9792c26ef8103ba21ac3/index.js#L16